bin/gem2rpm

```diff
@@ -9,7 +9,21 @@ require 'fileutils'
 require 'open-uri'
 require 'uri'
 
-options = Gem2Rpm::Configuration.instance.options
+options = begin
+  Gem2Rpm::Configuration.instance.options
+rescue Gem2Rpm::Configuration::ConfigurationError => e
```

Comment:
I followed current gem2rpm's programming style, though I do not think that the style is best.
You can see "Gem2Rpm::Template::TemplateError" in bin/gem2rpm.
And if you will check lib/gem2rpm/template.rb, you may see the "Gem2Rpm::Template::TemplateError" is used to throw the Error from template.rb.

So, I thought when throwing the Error from configuration.rb, the name "Gem2Rpm::Configuration::ConfigurationError" is not the best, but suitable with current system's style.

Of course if you like to catch the error "OptionParser::InvalidOption" in bin/gem2rpm, I can modify the code.

How do you think?  Do you still prefer to catch ""OptionParser::InvalidOption"" in bin/gem2rpm without creating new em2Rpm::Configuration::ConfigurationError?